actions/telemetry_actions.jsx

```diff
@@ -28,9 +31,13 @@ export function shouldTrackPerformance(state = store.getState()) {
 
 export function trackEvent(category, event, props) {
     Client4.trackEvent(category, event, props);
-    if (isDevMode() && category === 'performance' && props) {
-        // eslint-disable-next-line no-console
-        console.log(`${event}: ${props.duration}ms, fresh: ${props.fresh}`);
+    if (isDevMode() && (category === 'performance' || category === 'pageRequests') && props) {
+        const propKeys = Object.keys(props);
+        if (propKeys.length > 0) {
+            const propsAsString = propKeys.map((key) => `${key}: ${props[key]}`).join(', ');
+            // eslint-disable-next-line no-console
+            console.log(`${event} -> ${propsAsString}`);
```

Comment:
I also made a change like this in https://github.com/mattermost/mattermost-webapp/pull/9908, so that's definitely going to conflict 😛

Reply:
ha like minds :P